src/components/CategorySelector.tsx

```diff
@@ -93,19 +96,36 @@ export const CategorySelector = ({
           InputProps={{ endAdornment: <SearchIcon /> }}
           onChange={(event) => setSearchValue(event.target.value)}
         />
-        <Box
-          sx={{
-            display: 'flex',
-            gap: '0.5rem',
-            alignItems: 'center',
-          }}>
-          <FilterVintageIcon
+        {enableNviHighlightning ? (
+          <Chip
+            icon={
+              <FilterVintageIcon
+                color="primary"
+                titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
+                fontSize="small"
+              />
+            }
             color="primary"
-            titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-            fontSize="small"
+            onClick={() => setHighlightNviCategories(!highlightNviCategories)}
```

Comment:
Her er et forslag til løsning (som jeg ikke har sjekket at virker).
Mulig det burde være mulig å toggle av også, men kan også være det gir mening å bare slå på i dette tilfellet. Ikke noe med knappen som indikerer at alle vil velges ved trykk heller, så usikker på hvordan folk vil bruke dette 🤔 
```suggestion
            onClick={() => setSelectedCategories(...selectedCategories, ...nviApplicableTypes)}
```

Reply:
Måtte modifisere dette litt for å få det til å fungere. Ikke spesielt pent.